### ISSUE TITLE
Can't build

### ISSUE THREAD

**Issue by jacobhq** [nico-bachner/slidev-theme-geist#2]
My build fails with
```
'createElementBlock' is not exported by node_modules/slidev-theme-geist/node_modules/vue/dist/vue.runtime.esm-bundler.js, imported by node_modules/slidev-theme-geist/layouts/cover.vue
```

**Comment by nico-bachner** [comment]
This should be fixed in v0.8.0.

**Comment by jacobhq** [comment]
Thanks
